public/app/features/dashboard-scene/saving/provisioned/hooks.ts

```diff
@@ -27,7 +32,9 @@ export function useDefaultValues({ meta, defaultTitle, defaultDescription }: Use
   // Get config by resource name or folder UID for new resources
   const repositoryConfig = useGetResourceRepository({ name: annoName, folderUid: meta.folderUid });
   const repository = repositoryConfig?.spec;
-  const timestamp = Date.now();
+
+  const randomizer = newLetterRandomizer(); // replace with random
+  const timestamp = dateTime().format('YYYY-MM-DD-') + randomizer('xxxxx');
```

Comment:
```suggestion
  const timestamp = `${dateTime().format('YYYY-MM-DD')}-${randomizer('xxxxx')}`;
```

Reply:
Also I'd still reuse the timestamp from args, instead of crating one here.

Reply:
there is no timestamp in args -- just looks that way in the git diff :)